sql/moz-fx-data-marketing-prod/ga_derived/blogs_sessions_v2/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: Blogs Sessions V2
+description: |-
+  Intermediate table containing normalized sessions for blog.mozilla.org, sourced from Google Analytics 4 (GA4)
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived_ga4
```

Comment:
Is a separate DAG necessary? Why not just reused `bqetl_google_analytics_derived`?

Reply:
All the GA4 tables are going on the DAG bqetl_google_analytics_derived_ga4, since they all stem from a different source table than the old DAG bqetl_google_analytics_derived